### ISSUE TITLE
/ocm/send endpoint disappeared from Reva

### ISSUE THREAD

**Issue by michielbdejong** [pondersource/sciencemesh-nextcloud#166]
NC/OC-10 can no longer send shares in Network Beta because the /ocm/send endpoint was removed from Reva

**Comment by glpatcern** [comment]
The endpoint was renamed to `/sciencemesh/generate-invite`. By default it generates the invitation, and if an email is passed then it also sends the token, thus doing what the former `/ocm/send` used to do (despite not belonging to OCM strictly speaking)

**Comment by michielbdejong** [comment]
No, it's not the same, `/ocm/send` was an endpoint where the EFSS can tell reva to create an outgoing share and do an OCM call.

We need to move https://github.com/cs3org/reva/blob/v1.22.0/internal/http/services/ocmd/ocmd.go#L126-L128 and https://github.com/cs3org/reva/blob/v1.22.0/internal/http/services/ocmd/send.go to the sciencemesh service, I'll have a go at that tomorrow.
